src/main/resources/db/migration/V4_1__alter_dialogmeldingopplysninger_allownulls.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE DIALOGMELDINGOPPLYSNINGER
+ALTER COLUMN pasient_aktoer_id      DROP NOT NULL,
+ALTER COLUMN lege_aktoer_id         DROP NOT NULL;
```

Comment:
Hvor lagrer vi disse egentlig?

Reply:
Slik det er nå er det så vidt jeg skjønne er det veldig lite av dette som brukes i dag (bortsett fra evnt til manuell feilsøking). Men det blir mer aktuelt å bruke det når vi skal ha intern retry, for da må vi gjenskape den innkommende meldingen fra denne persisterte tilstanden.

Reply:
Men disse to aktuelle feltene må tåle null-verdier hvis vi skal persistere innkommende dialogmeldinger som feilet fordi vi ikke klarte å finne aktoer-id for pasient eller lege.

Reply:
Ja sant. For så lenge vi lagrer Fødselsnummer/D-nummer for disse, så trenger vi ikke lagre aktorId. AktorId vil vil helst unngå, så om vi kan kan få fjernet det helt fra padm2 hadde det vært flott.

Reply:
OK, får komme tilbake til dette i en senere PR 👍